updates/location_asset_mappings/v2.json

```diff
@@ -260,6 +260,181 @@
         "asset": "eip155:137/erc20:0x311434160D7537be358930def317AfB606C0D737",
         "location": "poloniex",
         "location_symbol": "NAKA"
+      },
+      {
+        "asset": "eip155:56/erc20:0xc27A719105A987b4c34116223CAE8bd8F4B5def4",
+        "location": "independentreserve",
+        "location_symbol": "ACE"
+      },
+      {
+        "asset": "eip155:1/erc20:0x91Af0fBB28ABA7E31403Cb457106Ce79397FD4E6",
+        "location": "iconomi",
+        "location_symbol": "AERGO"
+      },
+      {
+        "asset": "eip155:56/erc20:0x0E09FaBB73Bd3Ade0a17ECC321fD13a19e81cE82",
+        "location": "iconomi",
+        "location_symbol": "CAKE"
```

Comment:
did you add CAKE as common location mapping before? if yes then this shouldn't be needed

Reply:
@OjusWiZard i haven't added, as the kucoin was already there in v1.json